proto/hierarchy_prior.proto

```diff
@@ -31,6 +31,31 @@ message NNIGPrior {
   }
 }
 
+/*
+ * Prior for the parameters of the base measure in a Laplace - Normal Inverse Gamma hierarchy
+ */
+message LapNIGState {
+    double mean = 1;
+    double var = 2;
+    double shape = 3;
+    double scale = 4;
+    double mh_mean_var = 5;
```

Comment:
this is a question rather than a comment, also for @brunoguindani and @TeoGiane: do we prefer having the parameters of the proposal distribution in the state or in the `LapNIGPrior` message?

Reply:
Yes let's leave it like this